src/test/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,101 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private List<String> delimiterList = List.of(",", ":");
+
```

Comment:
계속 재사용할 목적이라면 필드에 존재하는 게 아니라 상수로 선언하는 것은 어떨까요?

```suggestion

public class StringCalculator {
    private static final List<String> delimiterList = List.of(",", ":");

```

또한, 네이밍을 복수형으로 변경해 보는 것은 어떨까요? 어떤 장점이 있을까요?? 
만약 List에서 Set으로 변경된다면 내부에서 사용하고 있는 부분을 모두 변경해야 하지 않을까요?

```suggestion

public class StringCalculator {
    private List<String> delimiters = List.of(",", ":");

```